libr/include/r_print.h

```diff
@@ -117,7 +117,7 @@ R_API void r_print_unset_flags(RPrint *p, int flags);
 R_API void r_print_addr(RPrint *p, ut64 addr);
 R_API void r_print_columns (RPrint *p, const ut8 *buf, int len, int height);
 R_API void r_print_hexii(RPrint *p, ut64 addr, const ut8 *buf, int len, int step);
-R_API void r_print_hexdump(RPrint *p, ut64 addr, const ut8 *buf, int len, int base, int step, int zoomsz);
+R_API void r_print_hexdump(RPrint *p, void *user, ut64 addr, const ut8 *buf, int len, int base, int step, int zoomsz);
```

Comment:
dont change this function signature, what you need from RCore* must be available in RPrint, in a callback or so